### PR TITLE
Updating protobuf and llvm hashes.

### PR DESCRIPTION
**Pull request by av8ramit**

### CODE REVIEW

**Comment by av8ramit** [comment]
Jenkins, test this please.

File: tensorflow/workspace.bzl [hunk 1]
```diff
@@ -484,10 +483,9 @@ def tf_workspace(path_prefix="", tf_repo_name=""):
   temp_workaround_http_archive(
       name = "llvm",
       urls = [
-          "http://mirror.bazel.build/github.com/llvm-mirror/llvm/archive/9889fe2290766430b99a2d4fadbc5ba92f8004b6.tar.gz",
           "https://github.com/llvm-mirror/llvm/archive/9889fe2290766430b99a2d4fadbc5ba92f8004b6.tar.gz",
       ],
-      sha256 = "00fb4a83a4dd1c046b19730a80e2183acc647715b7a8dcc8e808d49ea5530ca8",
+      sha256 = "a8da6d42ac7419e543a27e405f8b660f7b065e9ba981cc9cdcdcecb81af9cc43",
```
Comment (gunan) [inline tensorflow/workspace.bzl]: Also for llvm, let's keep the bazel mirror and remove the github url without updating the hash

**Comment by gunan** [comment]
```
17:09:19          -- SHA256 hash of
17:09:19              C:/tf_jenkins/home/workspace/tensorflow-pr-win-cmake-py/cmake_build/downloads/a6f29d8ac48d63293f845f2253eccbf86bc28321.tar.gz
17:09:19            does not match expected value
17:09:19              expected: '75d40ea8e68b0d1644f052fffe8f14a410b2a73d40ccb859a95c0578d194ec26'
17:09:19                actual: '1e40863d9f15dd6b15f8f18f49c500944be6118d9fe17e9dc58a1c709cadbb8a'
17:09:19          -- Hash mismatch, removing...
17:09:19          CMake Error at gemmlowp-stamp/download-gemmlowp.cmake:157 (message):
17:09:19            Each download failed!
```
Let's switch cmake build to only use our bazel mirrors.

File: tensorflow/workspace.bzl (hunk 1 above)
Comment (av8ramit) [reply tensorflow/workspace.bzl]: http://mirror.bazel.build/github.com/llvm-mirror/llvm/archive/9889fe2290766430b99a2d4fadbc5ba92f8004b6.tar.gz

Bazel mirror is a dead link.

Comment (gunan) [reply tensorflow/workspace.bzl]: @jart could you help us with this link?

Comment (gunan) [reply tensorflow/workspace.bzl]: OK, this link should now be working.

**Pull request merged**